### ISSUE TITLE
Unable to create a pipeline <Microsoft Fabric> <Data Engineer> <Microsoft Bug Bash>

### ISSUE THREAD

**Issue by marwashahid** [microsoft/studentambassadors#210]
### Describe the bug

I am creating a pipeline to ingest data. 

### What happened?

I am creating a pipeline to ingest data. I kept clicking on the new pipeline a lot of time and nothing happened. Afterwards a black small screen opened up. I waited each time but nothing happened. With the blank screen it becomes non-respondent to any mouse clicks as well. 

### What did you expect to happen?

I expected for input box to take the name of pipeline.

### How would you reproduce the bug?

1. In the Data Engineering tab , create a new lakehouse
2. Then click on new data pipeline
3. And the bug arrives

### Did this happen via Desktop and/or Smartphone?

Desktop

### Operating System

Windows 10

### What browsers are you seeing the problem on?

Chrome

### Version

latest

### Device

DELL Latitude E7250

### What is your current role?

Student

### What technical topic is this bug related to?

AI/Data Science/Machine Learning

### Relevant log output

```shell
It's an image and I do not have option to give image here.
```

```[tasklist]
### Tasks
```

**Comment by marwashahid** [comment]
It got created after the second login

**Comment by marwashahid** [comment]
I had to select Student Ambassador and accidently selected student.

**Comment by vivsridh4** [comment]
@marwashahid  can you confirm this issue has been resolved?

**Comment by marwashahid** [comment]
@vivsridh4  yes it got resolved and the pipeline worked smoothly afterwards.
